apps/src/javalab/Javalab.js

```diff
@@ -206,6 +207,17 @@ Javalab.prototype.init = function(config) {
       validation: getValidation(getStore().getState())
     }));
   }
+  if (this.isEditingExemplar) {
+    showLevelBuilderSaveButton(
+      () => ({
+        exemplar_sources: getSources(getStore().getState())
+      }),
+      'Levelbuilder: edit exemplar',
+      `/levels/${
+        getStore().getState().pageConstants.serverLevelId
+      }/update_exemplar_code`
+    );
+  }
 
   const startSources = config.level.lastAttempt || config.level.startSources;
```

Comment:
Side note: our initialization of sources is turning into a bit of a hot mess, might be worth cleaning this up a bit at some point.

Reply:
+1, mind making a tech debt item for that?

Reply:
Sure! https://codedotorg.atlassian.net/browse/JAVA-508